buildroot-external/rootfs-overlay/etc/mycroft/mycroft.conf

```diff
@@ -23,7 +23,7 @@
       "lang": "en-us"
     },
     "wake up": {
-      "module": "pocketsphinx",
+      "module": "ovos-ww-plugin-pocketsphinx",
```

Comment:
Saw this the other day, with changing toprecise plugin. I will include our plugin within the image.